templates/openstack/what-is-openstack.html

```diff
@@ -10,446 +13,556 @@
+            height="313",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"},) | safe
           }}
         </div>
         <div class="p-logo-section__item">
-          {{ image(url="https://assets.ubuntu.com/v1/2ec61a24-logo-deutsche-telekom.svg",
-                    alt="", width="144", height="144", hi_def=True, attrs={"class": "p-logo-section__logo"}, loading="lazy") | safe
+          {{ image(url="https://assets.ubuntu.com/v1/5c5ea603-cisco-logo.png",
+            alt="Cisco",
+            width="192",
+            height="313",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"},) | safe
           }}
         </div>
         <div class="p-logo-section__item">
-          {{ image(url="https://assets.ubuntu.com/v1/bc0dcda4-vmware-logo.svg",
-                    alt="", width="145", height="144", hi_def=True, attrs={"class": "p-logo-section__logo"}, loading="lazy") | safe
+          {{ image(url="https://assets.ubuntu.com/v1/a75dfa40-vmware-logo.png",
+            alt="VMware",
+            width="327",
+            height="313",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"},) | safe
           }}
         </div>
         <div class="p-logo-section__item">
-          {{ image(url="https://assets.ubuntu.com/v1/d8f890fb-logo-at%26t.svg",
-                    alt="", width="145", height="145", hi_def=True, attrs={"class": "p-logo-section__logo"}, loading="lazy") | safe
+          {{ image(url="https://assets.ubuntu.com/v1/68133ea9-AT&T-Logo.png",
+            alt="AT&T",
+            width="297",
+            height="313",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"},) | safe
           }}
         </div>
       </div>
     </div>
-    <div class="u-fixed-width">
-      <p>
-        Canonical is the founding member of the foundation, official maintainer
-        and publisher of OpenStack packages on Ubuntu, leader of the
-        <a href="https://docs.openstack.org/charm-guide/latest/">OpenStack Charms project</a>
-        and one of the biggest contributors to OpenStack all the time.
-      </p>
+    <div class="grid-row">
+      <div class="grid-col-start-large-5 grid-col-4 grid-col-start-medium-3 grid-col-medium-2">
+        <p>
+          Canonical is the founding member of the foundation, official maintainer
+          and publisher of OpenStack packages on Ubuntu, leader of the
+          <a href="https://docs.openstack.org/charm-guide/latest/">OpenStack Charms project</a>
+          and one of the biggest contributors to OpenStack all the time.
+        </p>
+      </div>
     </div>
   </section>
 
-  <section class="p-strip--light">
-    <div class="u-fixed-width">
+  {%- call(slot) vf_tiered_list(is_description_full_width_on_desktop=false, is_list_full_width_on_tablet=true) -%}
```

Comment:
```suggestion
  {%- call(slot) vf_tiered_list(is_description_full_width_on_desktop=false, is_list_full_width_on_tablet=false) -%}
```
Feel free to confirm with @mattea-turic, but I think this looks cleaner for med screens:
<img width="755" height="763" alt="Screenshot 2025-07-29 210917" src="https://github.com/user-attachments/assets/9fdb6125-0de0-4863-be14-dcb24c352291" />

vs demo:

<img width="769" height="726" alt="Screenshot 2025-07-29 210929" src="https://github.com/user-attachments/assets/add24b3e-d9c6-47c7-98be-443fe6f59bbe" />

Reply:
Thanks! I've updated them to your suggestion. @mattea-turic could you check if this is fine for medium screens?